transform-plugins/src/main/java/co/cask/hydrator/plugin/ValueMapperLookUp.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin;
+
+import co.cask.cdap.etl.api.Lookup;
+import co.cask.cdap.etl.api.LookupTableConfig;
+
+import java.util.Set;
+
+/**
+* Provides lookup functions for {@link ValueMapper}.
+* Used directly by {@link ValueMapper}.
+ */
+public class ValueMapperLookUp {
```

Comment:
As per Sree's suggestion, JavaScriptTransform class has been referred which has the same design structure for this class.  We thought that it must be for future use and has to be followed in same way.
However, delegate has been used in above process.
Config can be removed.
 Please confirm if we should keep or remove this class?